gluoncv/model_zoo/mask_rcnn/mask_rcnn.py

```diff
@@ -198,8 +222,12 @@ def __init__(self, features, top_features, classes, mask_channels=256, rcnn_max_
                              norm_layer=norm_layer, norm_kwargs=norm_kwargs)
             roi_size = (self._roi_size[0] * target_roi_scale, self._roi_size[1] * target_roi_scale)
             self._target_roi_size = roi_size
-            self.mask_target = MaskTargetGenerator(
-                self._batch_size, self._num_sample, self.num_class, self._target_roi_size)
+            if hasattr(mx.nd, 'mrcnn_target'):
```

Comment:
is it possible to put this check in to MaskTargetGenerator? so we don't have two class that do the same thing?

Reply:
I didn't want to change MaskTargetGenerator in order to leave it "usable" if one wants to try it and benchmark it against the fused op.

Reply:
Well, I can also add it through a `use_fused` arg

Reply:
what i meant is that inside masktargetgeneraotr, it will invoke fuse op if it is available, and use the original code if it is not available

Reply:
like https://github.com/dmlc/gluon-cv/blob/ec5bcece006120b6cbb2a7b420e00ae218083b44/gluoncv/nn/coder.py#L203

Reply:
Done